### ISSUE TITLE
Green Check Mark has Poor Color Contrast

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Closure-App#16]
The lime green check mark ( #A5EB78 ) to denote open locations on the "My Cities" screen has poor color contrast. The color contrast ratio is 1.42:1 on the white background ( #ffffff ).

You need to use a darker shade of green for it to pass. Plus, you might have trouble if you use the same shade on the cities page with the black text.

See screenshot below.

![closure notification on my cities screen's lime green check mark has poor color contrast](https://user-images.githubusercontent.com/10423487/36508586-2112a302-172b-11e8-9874-87d9d8a2f672.png)
